contents/docs/data/timestamps.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Let's add a link to this page from the /docs/data/events page

Reply:
Probably under this heading makes the most sense: https://posthog-haxsgxk6c-post-hog.vercel.app/docs/data/events#event-properties

Reply:
Actually, can you also do a search for the word "timestamp" in the docs and add an internal link there too if appropriate?